### ISSUE TITLE
Toolbar position

### ISSUE THREAD

**Issue by michaelbondar** [damienleroy/PowerAccent#103]
The toolbar with symbols needs to be closer to the cursor.

It would be nice, if the toolbar could be positioned 1 or 2 lines above the cursor. This would allow user not to move their eyes all over the screen looking for the toolbar.

**Comment by ChiediAStefano** [comment]
Exactly
In my case, with a 34" ultrawide monitor, fixed positions aren't useful. My eyes have to run all over the screen as I use multiple windows to write and chat. Yes, maybe after some time you'll get used and you already know which character is going to be pressed, but in the early use you have to look at the toolbar.

It should be implemented as Android, iOS, MacOS. A nice pop-up over the character for a quick glance.

**Comment by UrbanBreznik** [comment]
> Exactly In my case, with a 34" ultrawide monitor, fixed positions aren't useful. My eyes have to run all over the screen as I use multiple windows to write and chat. Yes, maybe after some time you'll get used and you already know which character is going to be pressed, but in the early use you have to look at the toolbar.
> 
> It should be implemented as Android, iOS, MacOS. A nice pop-up over the character for a quick glance.

Agreed at the end statement: "A nice pop-up over the character for a quick glance." .Yes yes and yes! And I know that this is a mac thing, but its so much more natural to "press and hold" the letter to get the popup than anything else. Its how it works on android an macs, just amazing. It could be just a optional way to trigger it.
![Accent positioning](https://user-images.githubusercontent.com/29950176/218442907-5b3b2647-39af-4f8b-86b4-fcd5bef12830.png)

**Comment by mrandreastoth** [comment]
Upvote.
